### PR TITLE
Improve colors

### LINKED ISSUE #386: Ensure that field colors in the App aren't too close
We need to tweak the color selection algorithm in the App so that label colors aren't as close as they are below:

(very hard to distinguish GT and predictions, which is the point of the colors!)

Related thoughts:
- should we hard code the colors to ensure this doesn't happen? Red, green, etc
- if we continue to randomize the colors, perhaps we should add an icon on the App that allows the colors to be regenerated if they're undesirable in some way

Screen Shot 2020-08-13 at 2 43 41 PM

### PR DESCRIPTION
**Pull request by lethosor**
## What changes are proposed in this pull request?

* Improved color selection
* A fixed color palette used for the first ~12 labels (fixes #386)
* A button to choose new colors

## How is this patch tested? If it is not, in a single sentence please explain why.

Local app instance

### Is this a user-facing change?

-   [ ] No. You can skip the rest of this section.
-   [x] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

App: Improved label colors 

### What areas of FiftyOne does this PR affect?

-   [x] `App`: FiftyOne application changes
-   [ ] `Build`: Build and test infrastructure changes
-   [ ] `Core`: Core `fiftyone` Python library changes
-   [ ] `Documentation`: FiftyOne documentation changes
-   [ ] `Other`

### Should this PR be mentioned in the release notes? If so, please choose on category:

-   [ ] `breaking-change` - The PR will be mentioned in the "Breaking Changes"
        section
-   [ ] `feature` - A new user-facing feature worth mentioning in the release
        notes
-   [ ] `bug-fix` - A user-facing bug fix worth mentioning in the release notes
-   [ ] `documentation` - A user-facing documentation change worth mentioning
        in the release notes

### CODE REVIEW

File: electron/app/utils/colors.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import _ from "lodash";
```
Comment (benjaminpkane) [inline electron/app/utils/colors.ts:1]: This is great, best of both worlds.

Comment (lethosor) [reply electron/app/utils/colors.ts:1]: Was this intended to be an approval or just a comment?

Comment (benjaminpkane) [reply electron/app/utils/colors.ts:1]: I was going to defer to @brimoor (specifically, the colors chosen) for approval. But I approve.

Comment (lethosor) [reply electron/app/utils/colors.ts:1]: The colors should be easy to change if we get a better list in the future

**Pull request merged**